components/apimgt/org.wso2.carbon.apimgt.rest.api.commons/src/main/java/org/wso2/carbon/apimgt/rest/api/common/impl/BasicAuthAuthenticator.java

```diff
@@ -63,13 +63,13 @@ public boolean authenticate(Request request, Response responder, ServiceMethodIn
                     return true;
                 }
             } else {
-                throw new APIMgtSecurityException("Missing Authorization header in the request.`",
-                        ExceptionCodes.INVALID_AUTHORIZATION_HEADER);
+                throw new APIMgtSecurityException("Missing 'Authorization : Basic' parameter in the request.`",
```

Comment:
Shouldn't it say header instead of parameter?
In the code comment above it says "If Basic auth header is not found returning true", but else part throws an exception?